zigzag/data/molecule_junit.xsd

```diff
@@ -45,7 +45,7 @@
     <!--Note: since sometimes either 'start_time' or 'end_time' might not be present because of setup/teardown
     failure we allow one less than the five possible test case properties.-->
     <xs:complexType name="testCasePropertiesType">
-        <xs:sequence minOccurs="4" maxOccurs="unbounded">
+        <xs:sequence minOccurs="0" maxOccurs="unbounded">
```

Comment:
Why was this changed to zero?

Reply:
I was thinking that was too restrictive, but on second thought, i guess that we'll need: test_step, test_id, jira, and start_time at least...